packages/dhis2/src/Adaptor.js

```diff
@@ -442,10 +475,17 @@ export function update(
  * });
  * @example <caption>all programs for an organization unit</caption>
  * get('programs', { orgUnit: 'TSyzvBiovKh', fields: '*' });
- * @example <caption>a single tracked entity instance by a unique external ID</caption>
- * get('trackedEntityInstances', {
- *   ou: 'DiszpKrYNg8',
- *   filter: ['flGbXLXCrEo:Eq:124', 'w75KJ2mc4zz:Eq:John'],
+ * @example <caption>a single tracked entity instance by a unique entity ID. Either `program`, `trackedEntityType` or `trackedEntities` should be specified in the params. See {@link https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#tracked-entities-get-apitrackertrackedentities TrackedEntities docs}</caption>
```

Comment:
I don't think this example is correct

Looking at the docs link you (helpfully!) provided, if I wanted to get a single tracked instance by id, I would do:
```
get('tracker/trackedEntities/F8yKM85NbxW')
```

(we later show this in another example, which is actually kinda confusing - which format should I use, and when?)

Regarding the required parameters... well I don't see that stuff mentioned in the docs and generally I don't think we should get into that. Definitely not in this example, and probably not at all in the `get()` docs. 

Because it's really complicated, we can never capture it all, and if we make mistakes we'll just confuse users. Users who want to use the REST API directly need to refer to the dhis2 docs and build the URL they need.

I think overhauling the docs is way out of spec for this work. All I'll ask for this PR as the new examples we provide are super accurate and useful.

Reply:
@josephjclark  The path you have displayed above still works